apps/crn-server/src/data-providers/users.data-provider.ts

```diff
@@ -38,13 +39,17 @@ import { buildEqFilterForWords, buildODataFilter } from '../utils/odata';
 
 export type CMSOrcidWork = OrcidWork;
 
-export interface UserDataProvider {
-  fetchById(id: string): Promise<UserDataObject | null>;
-  fetch(options: FetchUsersOptions): Promise<ListUserDataObject>;
-  create(input: UserCreateDataObject): Promise<string>;
-  update(id: string, update: UserUpdateDataObject): Promise<void>;
-}
-export class UserSquidexDataProvider implements UserDataProvider {
+export class UserSquidexDataProvider
+  implements
```

Comment:
you've already defined this type in here https://github.com/yldio/asap-hub/blob/18e2b18c373d25434b47d74493420d15c7e51391/apps/crn-server/src/data-providers/types/users.data-provider.types.ts#L10
so use that instead, same goes for all implementations